components/email/src/Email.svelte

```diff
@@ -234,7 +265,9 @@
         message_id: [message.id],
       }).then((results) => {
         results.forEach((msg: Message) => {
-          message.conversation = msg.conversation;
+          if (message) {
```

Comment:
Is this necessary since there's a check for `message` on line 262?

Reply:
Added this to resolve for a typescript error as the message type is defined to be `Message | undefined`.